docs/pyserini.md

```diff
@@ -1,19 +1,46 @@
 # Pyserini: Anserini Integration with Python
 
-Anserini was designed with Python integration in mind, for connecting with popular deep learning toolkits such as PyTorch. This is accomplished via [pyjnius](https://github.com/kivy/pyjnius). The `SimpleSearcher` class provides a simple Python/Java bridge, shown below:
+Anserini was designed with Python integration in mind, for connecting with popular 
+deep learning toolkits such as PyTorch. 
+[Pyserini](https://github.com/castorini/anserini/src/main/python/pyserini) 
```

Comment:
Our doc convention is a sentence on a line (although I break it sometimes also).
This allows the diffs to be easily scanned to see what changed.